libvast/vast/fbs/segment.fbs

```diff
@@ -17,14 +17,17 @@ table Segment {
   /// The version of the segment.
   version: Version;
 
+  /// The contained table slices.
```

Comment:
Maybe we should add a magic word before the version so we can check that the file actually belongs to us.

Reply:
Good thought, it's actually already there. Have a look at the bottom of the file: the `file_identifier` directive does that natively.

Reply:
I see.